KGTorrent/downloader.py

```diff
@@ -1,83 +1,71 @@
 """
-This module handles the actual download of Jupyter notebooks from Kaggle.
+This module defines the class that handles the actual download of Jupyter notebooks from Kaggle.
 """
 
 import logging
 import time
 from pathlib import Path
+from tqdm import tqdm
 
-import pandas as pd
 import requests
 from kaggle.api.kaggle_api_extended import KaggleApi
-from tqdm import tqdm
 
+# Imports for testing
```

Comment:
Is this comment still relevant?

Reply:
I wrote this comment over all the imports that are required exclusively for class testing. If it's superfluous, I delete them all.

Reply:
If they are not used anymore, you can delete them.

Reply:
They are used by testing procedures in each module that contain a class. I can delete them with the imports?

Reply:
Ah, OK. No, that's fine then, you can leave them. I thought that the code using the imports was no longer present.